src/main/resources/logback.xml

```diff
@@ -30,6 +30,7 @@
         </encoder>
     </appender>
 
+    <logger name="org.hibernate.engine" level="${CAF_HIBERNATE_LOG_LEVEL:-ERROR}" />
```

Comment:
- Let's change this to `CAF_LOG_LEVEL_HIBERNATE` so that if more come along that they can all start `CAF_LOG_LEVEL_`.
- Does this have to be here or can this come after the `<root>` node?